CRM/Core/I18n.php

```diff
@@ -719,6 +719,9 @@ public static  function getContactDefaultLanguage() {
     elseif ($language == 'current_site_language') {
       return CRM_Core_I18n::getLocale();
     }
+    else {
+      // If $language is none of the above, leave the selected language intact
+    }
```

Comment:
I don't think this is necessary?

The above conditions handle special values, and otherwise returns a specific languages.

Reply:
I don't really know what the hell I was thinking here. I think I left it while coding and forgot to take it out.